Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -1039,6 +1230,253 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
       return new Rect(r.x + padding, r.y + padding, r.width - (padding * 2), r.height - (padding * 2));
     }
 
+    /// <summary>
+    /// Returns a new Rect with the argument padding as an outward margin on each border
+    /// of this Rect; the result is a larger Rect.
+    /// </summary>
+    public static Rect PadOuter(this Rect r, float width) {
+      return new Rect(r.x - width, r.y - width,
+                      r.width + (width * 2f), r.height + (width * 2f));
+    }
+
+    #region Pad, No Out
+
+    public static Rect PadTop(this Rect r, float padding) {
```

Comment:
Not sure what exactly should happen, but I'd like all of these pad methods to have specifically defined edge cases where the padding is greater than a dimension.  Probably good to make sure the user doesn't end up with a rect with negative dimensions.

Reply:
le done